test/house_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/house.rb'
+require './lib/room.rb'
+
+class HouseTest < Minitest::Test
```

Comment:
We need you to be writing the tests BEFORE you implement functionality. You will not be able to pass this mod by writing tests after functionality. This pattern should be discontinued immediately.